extensions/transmission/src/utils/renderCells.tsx

```diff
@@ -1,6 +1,5 @@
 import { splitEvery } from "ramda";
 import BitField from "bitfield";
-import { renderToString } from "react-dom/server";
```

Comment:
Why isn't renderToString working here? It used to.

Reply:
It looks like it's a configuration/environment setup issue. https://stackoverflow.com/questions/66194269/typeerror-cannot-read-propertyreactcurrentdispatcherof-undefined

Reply:
I has already tested it. You might make better `package-lock.json`, but I failed with the package-lock.json file with the following error.
```
TypeError: Cannot read properties of undefined (reading 'ReactCurrentDispatcher')

react-dom-server-legacy.node.development.js:3634:0

---
3631:   }
3632: }
3633: 
3634: var ReactCurrentDispatcher = ReactSharedInternals.ReactCurrentDispatcher;
3635: var prefix;
3636: function describeBuiltInComponentFrame(name, source, ownerFn) {
3637:   {
---

node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:listTorrents.js:27539:9
__require:listTorrents.js:12:50
node_modules/react-dom/server.node.js:listTorrents.js:32976:11
```